README.md

```diff
@@ -102,7 +102,7 @@ const Example = () => {
           {
             id: "log-out",
             children: "Log out",
-            icon: "LogoutIcon",
+            icon: "ArrowLeftIcon",
```

Comment:
Maybe this is more appropriate?

```
ArrowRightOnRectangleIcon
```

Reply:
You are right @albingroen, I just made the change. Thanks!